applications/GeoMechanicsApplication/custom_conditions/Pw_normal_flux_condition.cpp

```diff
@@ -29,59 +30,39 @@ Condition::Pointer PwNormalFluxCondition<TDim, TNumNodes>::Create(IndexType
 }
 
 template <unsigned int TDim, unsigned int TNumNodes>
-void PwNormalFluxCondition<TDim, TNumNodes>::CalculateRHS(VectorType&        rRightHandSideVector,
+void PwNormalFluxCondition<TDim, TNumNodes>::CalculateRHS(Vector&            rRightHandSideVector,
                                                           const ProcessInfo& CurrentProcessInfo)
 {
-    // Previous definitions
-    const GeometryType&                             Geom = this->GetGeometry();
-    const GeometryType::IntegrationPointsArrayType& IntegrationPoints =
-        Geom.IntegrationPoints(this->GetIntegrationMethod());
-    const unsigned int NumGPoints = IntegrationPoints.size();
-    const unsigned int LocalDim   = Geom.LocalSpaceDimension();
+    const GeometryType&                             r_geometry = this->GetGeometry();
+    const GeometryType::IntegrationPointsArrayType& r_integration_points =
```

Comment:
What about using `auto` here?
```suggestion
    const auto& r_geometry = this->GetGeometry();
    const auto& r_integration_points =
```

Reply:
done